README.md

```diff
@@ -139,6 +139,49 @@ Linter-Specific Option | Description
 `suggestion`           | A string to be included in the rule's error message. Make this informative and specific to the rule that it is contained in.
 `addendum`             | A string to be included at the end of every error message of the rule set. (Optional)
 
+### ContentStyle
+
+Based on DeprecatedClasses, ContentStyle will find any words or phrases that
+violate the rule set that you provide.
+
+This `rule_set` is specified as a list of rules, each with a `violation` set and
+a corresponding `suggestion`. Optionally, you can also add a `case_insensitive:
+true` value to make ContentStyle ignore case when searching for violations.
+If your `violation` is a regex pattern, you can add a `regex_description` string
```

Comment:
I think `violation_description` or `pattern_description` is a better name for `regex description`.